Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionMicrosoftSysmon.yaml

```diff
@@ -1,19 +1,19 @@
 Parser:
   Title: Network Session Event ASIM parser for Sysmon (Event 3)
```

Comment:
@Alekhya0824 is this a duplicate PR?
https://github.com/Azure/Azure-Sentinel/pull/10571